core/trino-main/src/main/java/io/trino/server/TrinoSystemRequirements.java

```diff
@@ -60,6 +62,13 @@ private static void verifyJvmRequirements()
         verifyUtf8();
     }
 
+    private static void verifyManagementBeans()
+    {
+        if (!(ManagementFactory.getOperatingSystemMXBean() instanceof OperatingSystemMXBean)) {
```

Comment:
`io.trino.server.TrinoSystemRequirements#verifyFileDescriptor` is already making stronger guarantee that this is instance of `UnixOperatingSystemMXBean`.
Also, the return type here is already `OperatingSystemMXBean`, so not sure what the instanceof check will do

Reply:
`java.lang.management.OperatingSystemMXBean` vs `com.sun.management.OperatingSystemMXBean` (`extends java.lang.management.OperatingSystemMXBean`) :)